lib/mayaHydra/hydraExtensions/sceneIndex/mayaHydraMayaUsdProxyShapeSceneIndex.h

```diff
@@ -0,0 +1,111 @@
+//
+// Copyright 2023 Autodesk
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+#ifndef MAYA_HYDRA_MAYAUSD_PROXY_SHAPE_SCENE_INDEX_PLUGIN_H
+#define MAYA_HYDRA_MAYAUSD_PROXY_SHAPE_SCENE_INDEX_PLUGIN_H
+
+#if defined(MAYAHYDRALIB_MAYAUSD_IS_USED_TO_BUILD)
+//MayaHydra headers
+#include "mayaHydraLib/api.h"
+#include "mayaHydraLib/mayaHydra.h"
+
+//MayaUsdAPI headers
+#include <mayaUsdAPI/proxyStage.h>
+#include <mayaUsdAPI/proxyShapeNotice.h>
+
+//Usd/Hydra headers
+#include <pxr/base/tf/declarePtrs.h>
+#include <pxr/usdImaging/usdImaging/stageSceneIndex.h>
+#include <pxr/usdImaging/usdImaging/sceneIndices.h>// In USD 23.11+
+#include <pxr/imaging/hd/filteringSceneIndex.h>
+
+//Maya headers
+#include <maya/MObjectHandle.h>
+#include <maya/MMEssage.h>
```

Comment:
This won't compile on Linux, where the file system is case-sensitive.
```suggestion
#include <maya/MMessage.h>
```